src/Application.php

```diff
@@ -81,7 +110,12 @@ public function middleware(MiddlewareQueue $middlewareQueue): MiddlewareQueue
      */
     protected function bootstrapCli(): void
     {
-        parent::bootstrapCli();
-        $this->addPlugin('BEdita/I18n');
+        try {
```

Comment:
no `parent::bootstrapCli()` ?

Reply:
No `parent::bootstrapCli()` method exists. Previously we had it in `\BEdita\WebTools\BaseApplication` class.